.github/workflows/sitespeed.yml

```diff
@@ -0,0 +1,47 @@
+name: sitespeed.io
+on:
+  pull_request:
+    types: [closed]
```

Comment:
Could we run this action on opened PRs and identify performance issues before changes are merged?